apps/core/views.py

```diff
@@ -66,6 +66,14 @@ def get(self, request, *args, **kwargs):
                 ' GROUP BY core_omicsarea.name'
             ))
 
+            saved_analysis = Pixel.objects.raw((
+                "SELECT to_char(saved_at, 'YYYY,MM,DD') AS date,"
+                " count(*) AS nb,"
+                " 1 AS id"
+                " FROM core_analysis"
+                " GROUP BY date;"
```

Comment:
I already told you in another PR:

1. use the right Django model to perform the raw SQL query, you use `Pixel` but you should be using `Analysis`
2. uppercase SQL keywords like `COUNT`, `TO_CHAR`